### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by dmitry-zaitsev**

### CODE REVIEW

**Review by dangimatov — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,19 +1,79 @@
 # AnyMaps
-Lead Maintainer: [Dmitry Zaitsev](https://github.com/dmitry-zaitsev)
 
-### Usage / Description
+AnyMaps allows you to use the same API for different maps providers without the need to adjust existing Google Maps implementation (apart from changing the package name).
 
-```
+We are relying on this library in [car2go](https://www.car2go.com) to allow customers without Google Play Services to use our product.
+
+### Features
```
Comment (Diolor) [inline README.md]: h2

File: README.md [hunk 2]
```diff
@@ -1,19 +1,79 @@
...
+
+* Does not depend on any map provider - you can even use it without Google Maps.
+* Map provider can be changed at runtime.
+* Easy to extend. New providers can be added without changing the existing code.
+* Map providers supported out of the box: Google Maps, OpenStreet maps, Baidu maps.
+
+### Add it to your project
```
Comment (Diolor) [inline README.md]: h2

File: README.md [hunk 3]
```diff
@@ -1,19 +1,79 @@
...
+
+#### Step one
```
Comment (Diolor) [inline README.md]: h3

Comment (Diolor) [reply README.md]: and similar below

File: README.md [hunk 4]
```diff
@@ -1,19 +1,79 @@
...
 
-```
+We are relying on this library in [car2go](https://www.car2go.com) to allow customers without Google Play Services to use our product.
+
+### Features
+
+* Does not depend on any map provider - you can even use it without Google Maps.
+* Map provider can be changed at runtime.
+* Easy to extend. New providers can be added without changing the existing code.
+* Map providers supported out of the box: Google Maps, OpenStreet maps, Baidu maps.
```
Comment (Diolor) [inline README.md:15]: **Google Maps, OpenStreet maps, Baidu maps** ?

**Review by Diolor — APPROVED** [review]

**Pull request merged**